pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -435,7 +435,7 @@ func clearExpectations(t *testing.T, manager *daemonSetsController, ds *apps.Dae
 	}
 	sort.Strings(lines)
 	for _, line := range lines {
-		klog.Info(line)
+		klog.InfoS(line)
```

Comment:
this is not logging the info in structured form

Reply:
I'm not sure what to do here. @serathius what is the correct way to handle this?

Reply:
This is an test file, could we not use `klog`? Can you just use `t.Log()`